go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -85,6 +87,9 @@ func NewController(client clientset.Interface,
 	c.eventBroadcaster = broadcaster
 	c.eventRecorder = recorder
 
+	// repair controller
+	c.repair = NewRepair(5*time.Minute, st)
```

Comment:
What made you choose 5 minutes for the interval?

Reply:
nothing in particular, I had to start with something :smile: , 
but I've removed the periodic sync and it only repairs once now, at startup, the periodic loop didn't scale well, the latency to repair latency grows with the number of services, and it's racy (upstream has a bug with services due to this), and mainly because it will be odd that someone else create loadbalancer in ovn meanwhile the ovn controller is running, the current controller shouldn't left any dangling load balancer in the way that is designed
However I kept the code in case we need it in the future.